ci/coverage_publish.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Do not ever set -x here, it is a security hazard as it will place the credentials below in the
+# Travis logs.
+set -e
+
+if [ "${TRAVIS}" != "true" ]; then 
+  exit 0
+fi
+
+# available for master builds and PRs from originating repository (not forks)
+if [ "${TRAVIS_SECURE_ENV_VARS}" == "true" ]; then
+  mkdir -p ~/.aws
+  echo "[profile coverage]" >> ~/.aws/config
+  echo "aws_access_key_id=${COVERAGE_AWS_ACCESS_KEY_ID}" >> ~/.aws/config
+  echo "aws_secret_access_key=${COVERAGE_AWS_SECRET_ACCESS_KEY}" >> ~/.aws/config
+  echo "region=us-east-1" >> ~/.aws/config
+
+
+  COVERAGE_FILE=$(find -L ${ENVOY_BUILD_DIR} -name "coverage\.html" 2>&1 | grep "coverage\.html" | tail -n1)
```

Comment:
@htuch this a major hack but i couldn't find a symlink to get me to the build execroot from `$ENVOY_BUILD_DIR`. `bazel-envoy` works locally but not in travis. any ideas?